src/pages/messages/TasksPage.tsx

```diff
@@ -69,6 +69,10 @@ const StyledStatusRadio = styled(Radio)({
 
 const nviYearFilterValues = getNviYearFilterValues();
 
+interface LocationState {
+  previousSearch: string;
+}
+
 const TasksPage = () => {
   const { t } = useTranslation();
   const location = useLocation();
```

Comment:
Kan angi typen her, i stedet for `as` på linje 256
```suggestion
  const location = useLocation<LocationState | undefined>();
```